.gitlab/e2e/e2e.yml

```diff
@@ -308,6 +308,20 @@ new-e2e-cws:
   # Temporary, remove once we made sure the recent changes have no impact on the stability of these tests
   allow_failure: true
 
+new-e2e-discovery:
+  extends: .new_e2e_template
+  needs:
+    - !reference [.needs_new_e2e_template]
+    - deploy_deb_testing-a7_x64
+    - qa_agent
+    - qa_dca
```

Comment:
```suggestion
```

Unless I missed something, I think your test does not run the Docker version of the agent, so you should not need that one

Reply:
Thanks. I will remove these lines for now and add them back later when the Docker test is added. since I think we do want to test both Docker and non-Docker versions.